### ISSUE TITLE
Compatibility with non-PRO AMD Software

### ISSUE THREAD

**Issue by MathiasMagnus** [RadeonOpenCompute/rocm_smi_lib#127]
The HIP SDK on Windows (if one is not careful) will install the PRO version of the driver which installs `amdsmi.exe` which is nice. The issue is that Radeon Software will start arguing soon enough, that newer non-PRO drivers are available. Intalling it (because I also use my box for gaming) and having fixed the localization issue #126 by installing a new display language, it simply fails to enumerate my devices (IGP inside Ryzen 7 6800HS and the dGPU Radeon RX 6800S).

```
PS C:\Users\mate> amdsmi.exe
SMI-LIB has returned error '-1007' - 'Device Not found' Error code: -1007
PS C:\Users\mate> & 'C:\Program Files\AMD\ROCm\5.5\bin\hipInfo.exe' | sls Name:

Name:                             AMD Radeon RX 6800S
gcnArchName:                      gfx1032
Name:                             AMD Radeon(TM) Graphics
gcnArchName:                      gfx1035
```

Bottom line, I've not seen this utility work once, but it would be nice to have a `rocm-smi` alternative on Windows.

**Comment by dmitrii-galantsev** [comment]
Unfortunately we're keeping rocm-smi Linux only.
